sdk/storage/azure-storage-blob/azure/storage/blob/_lease_client_base.py

```diff
@@ -0,0 +1,33 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import uuid
+
+from typing import (  # pylint: disable=unused-import
+    Union, Optional, Any, IO, Iterable, AnyStr, Dict, List, Tuple, cast,
+    TypeVar, TYPE_CHECKING
+)
+
+if TYPE_CHECKING:
+    from datetime import datetime
+    from ._generated.operations import BlobOperations, ContainerOperations
+    BlobClient = TypeVar("BlobClient")
+    ContainerClient = TypeVar("ContainerClient")
+
+
+class BlobLeaseClientBase(object):
```

Comment:
Do we extract this class out just for consistency? what will happen if we leave this class inside _lease.py?